tests/foreman/cli/test_docker.py

```diff
@@ -391,6 +393,37 @@ def test_positive_delete_random_repo_by_id(self, module_org, module_target_sat):
             result = module_target_sat.cli.Repository.info({'id': repo['id']})
             assert result['product']['id'] in product_ids
 
+    def test_docker_upload_content(self, repo, module_org, module_target_sat):
+        """Create and sync a Docker-type repository, and attempt to run upload-content
+
+        :id: 031563fb-7265-44e3-9693-43e622b7756f
+
+        :Verifies: SAT-21359
+
+        :expectedresults: upload-content cannot be run with a docker type repository
+
+        :CaseImportance: Critical
+        """
+        assert int(repo['content-counts']['container-manifests']) == 0
+        module_target_sat.cli.Repository.synchronize({'id': repo['id']})
+        repo = module_target_sat.cli.Repository.info({'id': repo['id']})
+        assert int(repo['content-counts']['container-manifests']) > 0
+        remote_path = f'/tmp/{EXPIRED_MANIFEST}'
+        module_target_sat.put(DataFile.EXPIRED_MANIFEST_FILE, remote_path)
+        with pytest.raises(CLIReturnCodeError) as error:
+            module_target_sat.cli.Repository.upload_content(
+                {
+                    'name': repo['name'],
+                    'organization-id': module_org.id,
+                    'path': remote_path,
+                    'product-id': repo['product']['id'],
+                }
+            )
+        assert (
+            "Could not upload the content:\n  Cannot upload container content via Hammer/API. Use podman push instead.\n"
```

Comment:
```suggestion
            'Could not upload the content:\n  Cannot upload container content via Hammer/API. Use podman push instead.'
```

Reply:
this is to match more closely the reported text, there's probably a clean way to strip these newlines, but I didn't really bother.